package.json

```diff
@@ -18,7 +18,8 @@
     "dotenv": "^4.0.0",
     "jest": "^23.5.0",
     "nodemon": "^1.18.3",
-    "prettier": "1.14.2"
+    "prettier": "1.14.2",
+    "supertest": "^3.3.0"
```

Comment:
Do you mind if we use `got` instead of `supertest`? Supertest was a good option in the early days of Node, but with test runners like Jest, it's not quite as helpful anymore. I'd prefer if we use a simpler HTTP client like `got` alongside a utility like `test-listen` to get ephemeral URLs.

* `got` doesn't include its own assertion libraries, unlike `supertest`, letting us rely on Jest to provide that.
* `got` uses a more standard API for (a) sending post/patch/delete requests, rather than `supertest.send({})` (b) creating requests rather than supertest's more unique chaining approach
* `got` automatically throws on invalid responses (non-2xx status codes), so we only need to assert to test the body or if we want to test a particular status code 

Related, I think this will also help us avoid re-creating a server in every test, since we can pass around server or client instances:

```js
const http = require('http');
const got = require('got');
const listen = require('test-listen');
const handler = require('.');

const baseConfig = { /* get the env variables */ };

describe('micro-airtable-api', () => {
  describe('with all permissions', () => {
    const config = { ...baseConfig, allowedMethods: '*' };
    let server;
    let client;
    
    beforeAll(async () => {
      // By using `beforeAll` we only create a single server and just make different requests to it per-test
      server = http.createServer(handler(config));
      const baseUrl = await listen(server);
      client = got.extend({ baseUrl });
    });
    
    describe('GET /v0/Posts', () => {
      it('returns a successful response', async () => {
        // Got will automatically throw if the status code is an invalid response
        await client.get('/v0/Posts');
      });
    });
  });
});
```